### PR TITLE
Store token in session

### PR DESCRIPTION
**Pull request by mikefrancis**

### CODE REVIEW

File: src/Auth/Guard.php [hunk 1]
```diff
@@ -53,8 +53,16 @@ public function __construct($name, UserProvider $userProvider, Session $session,
      */
     protected function hasValidCredentials($user, $credentials)
     {
-        try {
+        if ($this->request->headers->has('Authorization')) {
+            $keys = ['token_type', 'access_token'];
+            $token = array_combine($keys, explode(' ', $this->request->headers->get('Authorization'), 2);
```
Comment (antpuleo2586) [inline src/Auth/Guard.php]: Is there a missing `)` at the end there?

Comment (mikefrancis) [reply src/Auth/Guard.php]: Yes 😔

File: src/Auth/Guard.php [hunk 2]
```diff
@@ -53,8 +53,16 @@ public function __construct($name, UserProvider $userProvider, Session $session,
      */
     protected function hasValidCredentials($user, $credentials)
```
Comment (antpuleo2586) [inline src/Auth/Guard.php]: Can these be type-hinted?

Comment (mikefrancis) [reply src/Auth/Guard.php]: Unfortunately not, it would need to be in the Laravel interfaces too.

Comment (antpuleo2586) [reply src/Auth/Guard.php]: Ah ok, damn

**Pull request merged**